pages/landing/enterprise-eks/index.html

```diff
@@ -0,0 +1,191 @@
+---
+layout: landing
+type: landing
+slug: index-page
+
+title: Enterprise apps on Amazon EKS. In about a day.
+excerpt: Build battle-tested Amazon EKS clusters, designed for enterprise with security, observability, and usability built-in. Manage 100% of your infrastructure using code, all in ~1 day. <b>Join now for special early access pricing!</b>
+permalink: /eks-for-enterprise/
+custom_js:
+- prism
+prism_css: /assets/css/prism-landing-zone.css
+
+technologies:
+- name: Spring Boot
+  logo: /assets/img/logos-technologies/logo-spring-boot.png
+- name: Ruby on Rails
+  logo: /assets/img/logos-technologies/logo-rails.png
+
+hero:
+  image: /assets/img/infrastructure-cube-icons/infrastructure-in-a-day.png
+  cta: Request early access
+  logos:
+  - /assets/img/logos-technologies/partner_logo_aws.png
+  - /assets/img/logos-technologies/partner_logo_kubernetes.png
+  - /assets/img/logos-technologies/partner_logo_terraform.png
+
+how_it_works:
+  - heading: "Step 1: Choose your EKS configuration"
+    precode: |
+      <figure><figcaption>eks-configuration.yml</figcaption>
```

Comment:
What is the thinking with a YML file here? Viewers of this will immediately wonder what it is that's processing the YAML file... Is it some custom Gruntwork thing? Some K8S YAML manifest thing? Something else?

My personal hunch is that users will prefer to do this via a web UI. A web form that lets you pick the exact settings shown here, but with lovely logos to click on (e.g., Prometheus logo, Istio logo, etc).

I suppose we could have two tabs here: "Configure via UI" and "Configure via code"...

Reply:
This was primarily because it was WIP. We added in the yaml because it was dead simple to construct and include for the basic skeleton. We were planning on adding in a form image as well, so it would be more like:

> I suppose we could have two tabs here: "Configure via UI" and "Configure via code"...

Reply:
In the interest of time/speed to delivery, I am going to defer this as a nice to have: https://github.com/gruntwork-io/gruntwork-io.github.io/issues/561